<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мастер обработки данных</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" rowspan="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Исходные данные</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="list_files"/>
          </item>
          <item row="1" column="1">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Преобразование</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <property name="topMargin">
             <number>10</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Ширина сегмента</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="tb_with">
              <property name="text">
               <string>1.0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="tb_beta">
              <property name="text">
               <string>1.0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Минимальная ширина пика (beta)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="topMargin">
             <number>10</number>
            </property>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="btn_run">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>Преобразовать</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>gui/icons/arrow-3-right.png</normaloff>gui/icons/arrow-3-right.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="0">
         <widget class="QStackedWidget" name="graph_widget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>240</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Предпросмотр</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progress">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Обработка данных</string>
    </property>
    <addaction name="action_run"/>
    <addaction name="action_apply"/>
    <addaction name="separator"/>
    <addaction name="action_back"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_back"/>
   <addaction name="action_apply"/>
   <addaction name="separator"/>
   <addaction name="action_run"/>
  </widget>
  <action name="action_run">
   <property name="icon">
    <iconset>
     <normaloff>gui/icons/arrow-3-right.png</normaloff>gui/icons/arrow-3-right.png</iconset>
   </property>
   <property name="text">
    <string>Преобразовать</string>
   </property>
   <property name="toolTip">
    <string>Преобразовать данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_back">
   <property name="icon">
    <iconset>
     <normaloff>gui/icons/arrow-1-backward.png</normaloff>gui/icons/arrow-1-backward.png</iconset>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
   <property name="toolTip">
    <string>Вернуться в главное окно</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_apply">
   <property name="icon">
    <iconset>
     <normaloff>gui/icons/check.png</normaloff>gui/icons/check.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить результат</string>
   </property>
   <property name="toolTip">
    <string>Сохранить результат преобразования</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_run</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_run</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_back</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_apply</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_files</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>select_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>run()</slot>
  <slot>apply()</slot>
  <slot>select_file()</slot>
 </slots>
</ui>
